path_config.go

```diff
@@ -227,6 +247,22 @@ func (b *jwtAuthBackend) pathConfigWrite(ctx context.Context, req *logical.Reque
 		}
 	}
 
+	// Validate response_types
+	if !strutil.StrListSubset([]string{responseTypeCode, responseTypeIDToken}, config.OIDCResponseTypes) {
+		return logical.ErrorResponse("invalid response_types %v. 'code' and 'id_token' are allowed", config.OIDCResponseTypes), nil
+	}
+
+	// Validate response_mode
+	switch config.OIDCResponseMode {
+	case "", responseModeQuery:
+		if config.hasType(responseTypeIDToken) {
+			return logical.ErrorResponse("query response_mode may not be used with an id_token response_type"), nil
+		}
+	case responseModeFormPost:
```

Comment:
Nit: might be nice to just add a `// Pass` or `// Success` comment on the next line down so maintainers don't think it's an oversight that there's no code following. Or alternatively, since there's only one case being checked now, could also just go to an `if` statement.